### ISSUE TITLE
How to test for accessibility with Cypress | Deque

### ISSUE THREAD

**Issue by studiomohawk** [frontendweekly/feeds#3831]
[How to test for accessibility with Cypress | Deque](https://www.deque.com/blog/how-to-test-for-accessibility-with-cypress/) / Let's review how to create test cases, integrate and use axe to check for accessibility violations with Cypress, and enhance accessibility tests.
